src/components/permissions/loginHelpers.ts

```diff
@@ -51,8 +50,16 @@ const buildTemplatePermissions = (templatePermissionRows: Array<PermissionRow>)
   return templatePermissions
 }
 
-const getSignedJWT = async (userInfo: UserInfo, templatePermissionRows: Array<PermissionRow>) => {
-  return await signPromise(compileJWT(userInfo, templatePermissionRows), config.jwtSecret)
+const getSignedJWT = async (
+  username: string,
+  userId: number,
```

Comment:
i kind of like the idea of userId and orgId being in an object, which would remove a few lines of code, and might make it a bit clearer. What do you think ?

Reply:
Yeah my plan is to have an object input for most of these helper methods and they check for which fields are defined for how to proceed. Will update this morning.